llm/anthropic_test.go

```
@@ -12,7 +12,7 @@ import (
 func TestNewAnthropicProvider(t *testing.T) {
 	key, ok := os.LookupEnv("ANTHROPIC_API_KEY")
 	if !ok {
-		t.Fatal("ANTHROPIC_API_KEY not found")
+		t.Skip("ANTHROPIC_API_KEY not found")
```

This really sets us up for unintentionally but silently skipping these tests when CI has a set up issue. If we're planning to run tests against anthropic APIs soon:tm:, please link a ticket here to revert this change when the full test setup is being done.

---

https://github.com/overmindtech/sdp-go/issues/323